### PR TITLE
[StructuralMechanicsApplication] Silence warning in python

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

~~~sh
/bin/Release/KratosMultiphysics/StructuralMechanicsApplication/element_sensitivity_domain_integration_process.py:59: SyntaxWarning: "is" with a literal. Did you mean "=="? 
~~~

**🆕 Changelog**

- [Silence warning in python](https://github.com/KratosMultiphysics/Kratos/commit/e16f0788ff2b0520441848a2a969dcdf7ce05b0d)

### CODE REVIEW

**Comment by loumalouomega** [comment]
@AlejandroCornejo pleass

File: applications/StructuralMechanicsApplication/python_scripts/element_sensitivity_domain_integration_process.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def __init__(self, model, parameter):
 
         # Get defined sub model parts of sensitivity model part as integration domains
         self.sensitivity_sub_model_parts = []
-        if len(sensitivity_sub_model_part_names) is 0:
+        if len(sensitivity_sub_model_part_names) == 0:
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/python_scripts/element_sensitivity_domain_integration_process.py]: Better remove the == 0 altogether 

Can find the pep for it later

**Review by philbucher — CHANGES_REQUESTED** [review]

File: applications/StructuralMechanicsApplication/python_scripts/element_sensitivity_domain_integration_process.py (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/python_scripts/element_sensitivity_domain_integration_process.py]: https://peps.python.org/pep-0008/

Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/python_scripts/element_sensitivity_domain_integration_process.py]:
```suggestion
        if not sensitivity_sub_model_part_names:
```
PEP8

Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/python_scripts/element_sensitivity_domain_integration_process.py]: Trivia: In Catalan/Valencian, Pep is a short for Joseph (Pepe in Spanish), and every time I see PEP8 I think in a song of my childhood:

https://www.comarcarural.com/valencia/musica/eltiopep.php

Muro is just the closest small village to my hometown

**Review by philbucher — APPROVED** [review]
:+1:

**Review by RiccardoRossi — COMMENTED** [review]
it does not look equivalent to me...for example your version woukd pass if it is a None

**Comment by loumalouomega** [comment]
I did as pep8 suggests

**Pull request merged**